README.md

```diff
@@ -0,0 +1,55 @@
+# Webots Animation Action
+
+This GitHub action creates a Webots animation of a simulation and publishes it to GitHub pages.
+
+<p align="center">
+  <img src="./assets/cover.png">
+</p>
+
+
+After each commit, Webots simulation will be recorded and published to `<username>.github.io/<repository>` as an X3D animation.
+In your browser, you can move around and zoom while the animation is playing.
+
+## Workflow
+
+Here is a simple GitHub workflow snippet which utilizes the action:
+```yaml
+name: Record animation
+
+on: [push]
+
+jobs:
+  record:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Record and deploy the animation
+        uses: cyberbotics/webots-animation-action@master
```

Comment:
Can you really reference a branch?
From my previous test I was only able to publish the action by creating a tag and an associated release (which we probably don't want to call `master` :wink:)
We should probably use version names such as the on from the official actions (e.g. https://github.com/marketplace/actions/checkout or https://github.com/marketplace/actions/setup-python).

Reply:
It works, see here :)
https://github.com/cyberbotics/webots-animation-template/tree/develop
I think you can put anything that can be associated with a commit, like hash, branch, or tag.

Once we are satisfied with API and implementation we can add a tag.

Reply:
Good to know, that's indeed a wonderful feature for the development! Then let's keep it like this for now and create the official 'release' to be integrated in the marketplace once it is fully ready.